src/hb-set.hh

```diff
@@ -707,7 +707,6 @@ struct hb_set_t
     void __next__ () { s->next (&v); if (l) l--; }
     void __prev__ () { s->previous (&v); }
     unsigned __len__ () const { return l; }
-    iter_t end () const { return iter_t (*s); }
```

Comment:
Removing it is not a good idea.  Just return `iter_t ()`.

Reply:
> Removing it is not a good idea. Just return `iter_t ()`.

I tried that, and returning iter_t () seems make "test-iter" test running forever